bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/SolarmanLoggerConfiguration.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarman.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link SolarmanLoggerConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Catalin Sanda - Initial contribution
+ */
+@NonNullByDefault
+public class SolarmanLoggerConfiguration {
+
+    /**
+     * Solarman Logger Thing Configuration Parameters
+     */
+    public String hostname = "";
+    public Integer port = 8899;
+    public String serialNumber = "";
+    public String inverterType = "sg04lp3";
+    public int refreshInterval = 30;
+    @Nullable
+    public String additionalRequests = "";
```

Comment:
```suggestion
    @Nullable
    public String additionalRequests;
```

Why set a default value if it is nullable? Either remove the default or the @nullable annotation